addons/project/controllers/portal.py

```diff
@@ -244,6 +244,8 @@ def _task_get_page_view_values(self, task, access_token, **kwargs):
             'project_accessible': project_accessible,
             'task_link_section': [],
         }
+        if project:
+            values.update(project_id=project.id)
```

Comment:
I learned the update is slower than this:
```suggestion
            values['project_id'] = project.id
```
when there are only one item to update in the dict. 🙂

Reply:
I didn't know, thanks! Fixed